app/utils/questionsManagement.js

```diff
@@ -1,5 +1,3 @@
-import _ from 'lodash';
-
 import {
```

Comment:
Please add unit tests for the fetcher. This important component that is easy to break.

Reply:
added